### PR TITLE
Added date filter.

### PR DESCRIPTION
**Pull request by priya-actonate**
@vbiarulkumar Please review

### CODE REVIEW

File: src/controls/ValueEditor.tsx [hunk 1]
```diff
@@ -15,75 +15,133 @@ const ValueEditor: React.FC<ValueEditorProps> = ({
...
 
-  const onSelectChange = (value:any) => { handleOnChange(value.value)};
-  const onTextInputChange = (e:any) => { handleOnChange(e.target.value)};
-  const onTextAreaChange = (e:any) => { setValue(e.target.value)};
-  const applyTextAreaChange = () => { handleOnChange(_value)};
-  const onCheckboxChange = (e:any) => handleOnChange(e.target.checked);
-  const  onAutoSuggetionChange = (value:any) => {
-      handleOnChange(value);
+  const onSelectChange = (value: any) => {
+    handleOnChange(value.value);
```
Comment (vbiarulkumar) [inline src/controls/ValueEditor.tsx:29]: Remove unwanted  line spaces

**Review by vbiarulkumar — APPROVED** [review]
reviewed

File: src/query-builder.less [hunk 2]
```diff
@@ -11,342 +11,1044 @@
...
 }
 .queryBuilder-footer-advanced {
-    .queryBuilder-footer-title {
-        padding: 4px 6px;
+  .queryBuilder-footer-title {
+    padding: 4px 6px;
+  }
+}
+
+.date-filter-wrapper {
```
Comment (vbiarulkumar) [inline src/query-builder.less]: keep it calendar css in separate file, or put it into rule block

Comment (priya-actonate) [reply src/query-builder.less]: done

**Review by vbiarulkumar — APPROVED** [review]

**Pull request merged**